### ISSUE TITLE
Make it a kubectl plugin

### ISSUE THREAD

**Issue by luisdavim** [robusta-dev/krr#197]
Hi, this is not a real issue but it would be nice to be able to install this tool via `krew` as a kubectl plugin, I think that would also help users discovering this tool.

**Comment by aantn** [comment]
Good idea. Can't commit to a timeline on this at the moment, but if anyone wants to contribute it would be welcome!
